docs/Netzwerke/Webserver/apis.md

```diff
@@ -0,0 +1,53 @@
+Server: A powerful computer that runs an API
+
+API: The "hidden" portion of a website that is meant for computer consumption
+
+Client: A program that exchanges data with a server through an API
+
+A protocol is the rules that computers have while talking to each other.
+
+Hyper-Text Transfer Protocol=HTTP
+
+It is easier that everybody uses HTTP so developers only have to learn that API.
+
+To make a valid request, the client needs to include four things:
+```
+URL (Uniform Resource Locator) 1   
+URLs are an easy way for the client to tell the server which resources it wants to interact with.
+
+Method
+GET - Asks the server to retrieve a resource
+POST - Asks the server to create a new resource
+PUT - Asks the server to edit/update an existing resource
+DELETE - Asks the server to delete a resource
+
+List of Headers
+Headers provide meta-information about a request. They are a simple list of items like the time the client sent the request and the size of the request body.
+
+
+Body
+The request body contains the data the client wants to send the server.
+```
+
+HTTPS responds are similar to request simply instead of a url and a method it has "Status Codes"
```

Comment:
@TheTahm Nope, the `s` in HTTPS stands for "secure", not for status code. Both, HTTP and HTTPS have status codes